demo/crypto-perf/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM centos:7
+
+ENV http_proxy http://proxy.ir.intel.com:911/
+ENV https_proxy http://proxy.ir.intel.com:911/
+ENV no_proxy http://proxy.ir.intel.com:911
+ENV SHELL=/bin/bash
+COPY ./resolv.conf /etc/resolv.conf
+RUN rpm --rebuilddb
+RUN yum -y clean all; yum -y update; RUN yum -y install epel-release  httpd httpd-tools curl traceroute tcpdump bind-utils nc net-tools nmap screen htop cairo-devel hwloc-libs hwloc-gui pciutils
+RUN mv /usr/sbin/tcpdump /usr/bin/tcpdump
+WORKDIR /usr/src/dpdk
+COPY . /usr/src/dpdk
+WORKDIR /usr/src/dpdk/build/app
```

Comment:
I suspect there's a risk this demo is going to work only if DPDK has been built on the host running CentOS 7.
I'd build DPDK as a part of Docker image creation.

Reply:
I'll update this, Thanks for your valuable comments